tst/EnergyPlus/unit/UnitarySystem.unit.cc

```diff
@@ -8728,7 +8728,7 @@ TEST_F(EnergyPlusFixture, UnitarySystemModel_ASHRAEModel_WaterCoils)
     // these next 2 variables are used to modulate the coil PLR irrespective of the fan PLR - they are non-zero when the model is called and CAN be 0
     // when load exceeds capacity the ASHRAE model is the only model that uses these variables, and flow is determined by Heat/CoolWaterFlowRatio *
     // max other models will show 0 here and in this case water flow will equal max flow * PartLoadRatio
-    EXPECT_NEAR(thisSys->HeatCoilWaterFlowRatio, 0.01374, 0.0001); // heating coil water flow ratio, heating coil is on
+    EXPECT_NEAR(thisSys->HeatCoilWaterFlowRatio, 0.04123, 0.0001); // heating coil water flow ratio, heating coil is on
```

Comment:
I spent some time looking at what caused this. All conditions are the same, inlet air temp and flow, inlet water temp and flow. The water heating coil now goes through the CalcAdjustedCoilUA function differently because the WaterCoil().DesAirMassFlowRate is now > 0. And because of that the if is true whereas the else was previously executed. Which causes a different UA and then a different coil outlet air temp. It looks like it's working correctly now. I'll have to check why WaterCoil().DesAirMassFlowRate was previoulsy 0.

    if ((AirConvectTerm > 0.0) && (WaterConvectTerm > 0.0)) {
        state.dataWaterCoils->WaterCoil(CoilNum).UACoilVariable = 1.0 / ((1.0 / WaterConvectTerm) + (1.0 / AirConvectTerm));
    } else {
        // use nominal UA since variable UA cannot be calculated
        state.dataWaterCoils->WaterCoil(CoilNum).UACoilVariable = state.dataWaterCoils->WaterCoil(CoilNum).UACoil;
    }